.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -7,70 +7,63 @@ step: 3
 name: Week 1 Step 3
 ---
 
-## Week 1 Step 3
+Week 1 Step 3 ⬤⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 10-15 minutes
 
-🕐 Estimated completion: 10-15 minutes
+## Deliverin' Cat Pics with Postman? 
+It is important to test your code as you write it. Postman is a great tool when trying to dissect RESTful APIs made by others or test ones you have made yourself. It offers a sleek user interface to make HTML requests, without the hassle of writing a bunch of code just to test your API.
 
-### Postman, APIs, and requests
+*In this step, you will be using Postman to test the [Cataas API](cataas.com) by sending a GET request*
 
-<details>
-<summary>:question: What is Postman and why do we need it?</summary>
-  </br>
+You can sign up for Postman [here](https://www.postman.com/)
 
-Later, when we begin to code our Azure Function, we are going to need to test it. 
+### ✅  Task:
 
-**How?** Just like our final web app and the first function we will code, we'll be sending requests to the Function's endpoint.
+- [ ]  With Postman, send a GET request to the Cat Picture API and receive a cat picture with "Serverless" written on it 
```

Comment:
You're asking them to return "serverless" but the endpoint for the API you provided is: https://cataas.com/cat/cute/says/Bitcamp
the "Bitcamp" in the endpoint tells the API endpoint what to print on the picture